inspirehep/modules/workflows/tasks/manual_merging.py

```diff
@@ -0,0 +1,169 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+"""Tasks related to manual merge generic record."""
+
+from __future__ import absolute_import, division, print_function
+
+from inspire_json_merger.inspire_json_merger import inspire_json_merge
+from inspire_dojson.utils import get_record_ref
+from inspirehep.modules.records import InspireRecord
+from inspirehep.utils.record_getter import get_db_record, RecordGetterError
+from inspirehep.modules.workflows.utils import with_debug_logging
+from invenio_db import db
+from workflow.engine_db import WorkflowStatus
+
+
+def _check_sources(wf_obj, head, update):
```

Comment:
This is going to go away. It was just a temporary fix, now made in inspire-json-merger`. :+1: